app/assets/stylesheets/notifications.scss

```diff
@@ -410,5 +410,9 @@
 
   .load-more-wrapper {
     @include load-more;
+
+    button {
+      margin-top: 0;
```

Comment:
after having moved the "load more" button outside the notification list container we have too much margin